### ISSUE TITLE
dark preference ui

### ISSUE THREAD

**Issue by ballinger** [ltGuillaume/FeedMei#6]
hi, is that by intention that the profile/ preference page uses the white theme even tough the dark theme is set - or is something not working... thanks!

**Comment by ltguillaume** [comment]
Nope, you're right: the dark theme is only applied to the main UI. I didn't feel like it was worth the time to get the dark mode for Preferences working right, too, primarily because I never ever use the dark mode myself anyway, but also because it's not exactly a place where you come often.
